### PR TITLE
ADDED: School Admission Form updated

### PR DESCRIPTION
**Pull request by surendran18022002**
All review points are  updated

### CODE REVIEW

File: Surendran/School Admission Form/js/intex.js [hunk 1]
```diff
@@ -1,127 +1,176 @@
...
+ {
+        var obj = localStorage.getItem("txtvalue");
+        var arr = JSON.parse(obj);
+        var delrow = localStorage.getItem("row");
+        var row = JSON.parse(delrow);
+      
+        if (confirm("delete this application"))
+        {
+               if(arr[0]==null)
+               {
```
Comment (NavinAT) [inline Surendran/School Admission Form/js/intex.js:93]: check indent

File: Surendran/School Admission Form/intex.html
Comment (NavinAT) [inline Surendran/School Admission Form/intex.html:1]: 1. Check alignment
image
2. Use the input element style for select element
3. 'medium' - Medium
4. Should validate gender, it should not allows to save None

File: Surendran/School Admission Form/output.html
Comment (NavinAT) [inline Surendran/School Admission Form/output.html:1]: 1. Header text should not wrap

image

Bug in Edit page please see the video:

https://user-images.githubusercontent.com/45414603/225608585-4d38f695-3996-42cf-80ba-b791a5d59419.mp4

Value got removed in table

**Review by NavinAT — CHANGES_REQUESTED** [review]

File: Surendran/School Admission Form/js/intex.js [hunk 2]
```diff
@@ -1,127 +1,176 @@
...
+}
+ function teacher_login()
+ {
+        window.location.href = "output.html";
+        row=0;
+        localStorage.setItem("row", JSON.stringify(row));
+}
+/*intex page */
+function getdata(){
+        var std     = document.getElementById("std").value;
```
[the remark this answers is not in the document]
Comment (surendran18022002) [reply Surendran/School Admission Form/js/intex.js]: completed

File: Surendran/School Admission Form/js/intex.js [hunk 3]
```diff
@@ -1,127 +1,176 @@
...
+        var cty     = document.getElementById("cty").value;
+        var dt      = document.getElementById("dt").value; 
+        var st      = document.getElementById("st").value; 
+        var pc      = document.getElementById("pc").value;    
+        var mf      = document.getElementById("mf").value;
+        var arrobj=[];
+        var obj = localStorage.getItem("txtvalue");  
+        let text={"std": std,"sec": sec,"medium": medium, "fname": fname,"mname": mname,"lname": lname, "DOB": DOB,"age": age,farname :farname,momname: momname ,"bldgp": bldgp ,"mrf": mrf,"adds": adds, "adds2":adds2, "cty": cty, "dt": dt,"st":st, "pc": pc, "mf":mf};
+        if(obj)
+              {
```
[the remark this answers is not in the document]
Comment (surendran18022002) [reply Surendran/School Admission Form/js/intex.js]: completed

File: Surendran/School Admission Form/intex.html
Comment (surendran18022002) [reply Surendran/School Admission Form/intex.html:1]: completed

File: Surendran/School Admission Form/output.html
Comment (surendran18022002) [reply Surendran/School Admission Form/output.html:1]: completed

**Pull request merged**